.github/workflows/build.yml

```diff
@@ -0,0 +1,47 @@
+name: Build
+on:
+  create:
+    tags:
+       - 'v*.*'
+    
+jobs:
+  docker_build_push:
+    env:
+      GIT_REF: ${{ github.ref }}
+    if: startsWith(github.ref, 'refs/tags/v')
+
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Scala
+      uses: actions/setup-java@v2
+      with:
+        distribution: adopt
+        java-version: 11
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Show output of sed to build.sbt command
+      run: |
+        VERS_TAG=$(echo $GIT_REF | sed 's/refs\/tags\/v//g')
+        VERS_TAG="\"$VERS_TAG"\"
+        sed "s/version := *.*/version := $VERS_TAG/" build.sbt
+    - name: Write Tag to Scala Build Version
+      run: |
+        VERS_TAG=$(echo $GIT_REF | sed 's/refs\/tags\/v//g')
+        VERS_TAG="\"$VERS_TAG"\"
+        sed -i "s/version := *.*/version := $VERS_TAG/" build.sbt
+    - name: Login to DockerHub
+      uses: docker/login-action@v1 
+      with:
+         username: ${{ secrets.DOCKERHUB_USERNAME }}
+         password: ${{ secrets.DOCKERHUB_TOKEN }}
+    - name: Build
+      run: sbt docker:publishLocal
```

Comment:
Think you could change the configuration of the docker container so that it's possible to start the container running with a bash terminal (instead of it forcing us to always run the application). 

[dockerEntrypoint](https://www.scala-sbt.org/sbt-native-packager/formats/docker.html#environment-settings) Seems like it might do the job.

We need this so that Jenkins can just spin up the container (and execute the right command as many times as it needs to).

Reply:
For clarification - is the intended use that Jenkins can interact with an existing Docker container? So it spins up a container from an image and then queries it from bash, reaching into the Docker container telling it to run a certain script as many times as is needed?

So
1. Load up container from image created during workflow build script.
2. Interact with container's bash prompt, calling a script.
3. Do some other stuff for now.
4. Go back to 2. unless the container is not running, in which case go back to 1.

?

Reply:
Yes, so Jenkins will spin up the container and then execute bash scripts as many times as it requires.

You should also be able to run the docker container locally with something like `docker run -it --rm gsscogs/csvw-check bash` (or replace `bash` with `sh`) and that should allow you to run anything you like inside the container.

Reply:
Had some frustrations with this - I can't seem to determine where the entry point is currently being set/overridden from. 

I would suggest a fresh set of eyes on this one.

For now, the other features of Docker should probably cover this one.
- ```docker run -it --entrypoint /bin/bash csvw-check:latest``` successfully starts a bash terminal in the container that's interactive as a console.
- ```docker run -it -d --name=checker --entrypoint /bin/bash csvw-check:latest``` can be used to deploy a detached container and then a subsequent ```docker exec -it checker /bin/bash``` can be done repeatedly do execute commands into a bash terminal.
- Failing this, the standard run command runs a container _fast_ and likely provides the simplest and cleanest implementation with Jenkins.

Reply:
Resolved and tested with @robons b0c2f8c commit.

Reply:
@robons  I've committed and pushed 9e65b9f into this branch to add csvw-check to path. When you get a moment, can you please test it and then merge this pull request if there are no other outstanding issues?

Reply:
Looks good to me.